doc/setup_instructions.md

```diff
@@ -408,7 +408,6 @@ installer release. After installing, add the installed folder (where
 in command prompt or powershell, it should print 7zip version info and
 command line usage.
 
-
```

Comment:
I think that keeping this blank line makes the file a bit more readable in raw text to mark a section ending instead of just a paragraph.